src/chainlit/frontend/src/components/selectInput.tsx

```diff
@@ -0,0 +1,189 @@
+import _ from 'lodash';
```

Comment:
Try to not import the whole lodash library, instead you can do 
`import get from 'lodash/get';`. This will result in a much smaller bundle

Reply:
Still not resolved

Reply:
My bad, forgot it! But actually this was an unused import, maybe we should add a rule for that 👀